packages/jest-cli/src/SearchSource.js

```diff
@@ -42,6 +42,11 @@ type PatternInfo = {
   testPathPattern?: string,
   input?: string,
   shouldTreatInputAsPattern?: boolean,
+  lastCommit?: boolean,
```

Comment:
can you alphabetize the values here? We usually try to order object keys based on the alphabet.